django_perf_rec/orm.py

```diff
@@ -41,18 +46,32 @@ def patch_QuerySet():
     """)
 
 
-def patch_Query():
-    patchy.patch(Query.add_extra, """\
-        @@ -13,7 +13,7 @@
-                     param_iter = iter(select_params)
-                 else:
-                     param_iter = iter([])
-        -        for name, entry in select.items():
-        +        for name, entry in sorted(select.items()):
-                     entry = force_text(entry)
-                     entry_params = []
-                     pos = entry.find("%s")
-    """)
+if DJANGO_VERSION < StrictVersion("2.2"):
```

Comment:
The comparison should be for the new version first and you can use `django.VERSION` more simply like

```python
if django.VERSION > (2, 2):
```

Reply:
done